spec/truffle/interop/from_java_array_spec.rb

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved. This
+# code is released under a tri EPL/GPL/LGPL license. You can use it,
+# redistribute it and/or modify it under the terms of the:
+#
+# Eclipse Public License version 1.0
+# GNU General Public License version 2
+# GNU Lesser General Public License version 2.1
+
+require_relative '../../ruby/spec_helper'
+
+guard -> { !Truffle.native? } do
+  describe "Truffle::Interop.from_java_array" do
+
+    it "converts a byte[]" do
+      array = Truffle::Interop.java_type("byte[]").new(3)
+      array[0] = 1
+      array[1] = 2
+      array[2] = 3
+      array = Truffle::Interop.from_java_array(array)
+      array[0].should == 1
+      array[1].should == 2
+      array[2].should == 3
+      array.size.should == 3
```

Comment:
These 4 lines could just be `array.should == [1, 2, 3]` since it's a Ruby array (same below).

Reply:
Fixing.